src/main/java/Sunday.java

```diff
@@ -0,0 +1,33 @@
+import command.Command;
+import exceptions.SundayException;
+import utilities.Parser;
+import utilities.Ui;
+
+public class Sunday {
+    private Ui ui;
+    public Sunday(String filepath) {
+        this.ui = new Ui();
+        try {
+            Command.INITIALIZE.execute("data/sunday.txt");
+        } catch (SundayException e) {
+            Ui.printException(e);
+        }
+    }
+
+    private void run() {
+        Command command = null;
+        while (command != Command.BYE) {
+            try {
+                String[] fullCommand = ui.readCommand();
+                command = Parser.parse(fullCommand);
+                command.execute(fullCommand[1]);
+            } catch (SundayException e) {
+                Ui.printException(e);
+            }
+        }
+        this.ui.close();
+    }
+    public static void main(String[] args) {
```

Comment:
Line break i think could be inserted here to improve readability

Reply:
You're right! I've added line breaks everywhere since this commit to improve readability. Thanks!